### PR TITLE
First pass at a Yeoman Generator

### PR DESCRIPTION
**Pull request by stjohnjohnson**
This Yeoman generator creates a new npm module pre-loaded with:
 - License
 - Readme with a variety of badges
 - Contributing doc
 - JSCS config
 - Wercker config with CD support
 - Author default setup as you

### CODE REVIEW

**Comment by petey** [comment]
We can use just jscs, we don't need jshint stuff.

File: app/templates/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "contributors": [
+    "Dao Lam <d2lam@yahoo-inc.com>",
+    "Darren Matsumoto <bobafett@yahoo-inc.com>",
+    "Jeremiah Wuenschel <jer@yahoo-inc.com>",
+    "Noah Katzman <nkatzman@yahoo-inc.com>",
+    "Peter Peterson <petey@yahoo-inc.com>",
+    "St. John Johnson <stjohn@yahoo-inc.com>",
+    "Tiffany Kyi <tkyi@yahoo-inc.com>"
+  ],
+  "jshintConfig": {
```
Comment (petey) [inline app/templates/package.json]: This isn't necessary. It is only for internal tools.

File: app/templates/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "bugs": "https://github.com/screwdriver-cd/<%= name %>/issues",
+  "keywords": [
+    "screwdriver",
+    "yahoo"
+  ],
+  "license": "BSD-3-Clause",
+  "author": "<%= author %> <<%= email %>>",
+  "contributors": [
+    "Dao Lam <d2lam@yahoo-inc.com>",
+    "Darren Matsumoto <bobafett@yahoo-inc.com>",
```
Comment (FenrirUnbound) [inline app/templates/package.json]: Preference is my github.com username plz 😺 
`Darren Matsumoto <aeneascorrupt@gmail.com>`

File: app/templates/package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "Jeremiah Wuenschel <jer@yahoo-inc.com>",
+    "Noah Katzman <nkatzman@yahoo-inc.com>",
+    "Peter Peterson <petey@yahoo-inc.com>",
+    "St. John Johnson <stjohn@yahoo-inc.com>",
+    "Tiffany Kyi <tkyi@yahoo-inc.com>"
```
Comment (tkyi) [inline app/templates/package.json]: tiffanykyi@gmail.com please

File: app/templates/package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "yahoo"
+  ],
+  "license": "BSD-3-Clause",
+  "author": "<%= author %> <<%= email %>>",
+  "contributors": [
+    "Dao Lam <d2lam@yahoo-inc.com>",
+    "Darren Matsumoto <bobafett@yahoo-inc.com>",
+    "Jeremiah Wuenschel <jer@yahoo-inc.com>",
+    "Noah Katzman <nkatzman@yahoo-inc.com>",
+    "Peter Peterson <petey@yahoo-inc.com>",
```
Comment (petey) [inline app/templates/package.json]: Same for me `jedipetey@gmail.com`

**Comment by stjohnjohnson** [comment]
@tkyi @petey @FenrirUnbound Updated.

**Comment by FenrirUnbound** [comment]
👍 

PR looks good as a first start 😸

**Comment by stjohnjohnson** [comment]
@jer are you okay with this email address?

**Comment by jer** [comment]
👍

**Pull request merged**